.github/workflows/labeller.yml

```diff
@@ -2,26 +2,12 @@ name: Labeller
 
 on:
   issues:
-    types:
-      - opened
-      - labeled
-      - unlabeled
+    types: [ opened, reopened, labeled, unlabeled ]
   pull_request_target:
-    types:
-      - opened
-      - labeled
-      - unlabeled
+    types: [ opened, reopened, labeled, unlabeled ]
 
 jobs:
   label:
-    runs-on: ubuntu-latest
-    steps:
-
-      - uses: puppetlabs/community-labeller@v1.0.1
-        name: Label issues or pull requests
-        with:
-          label_name: community
-          label_color: '5319e7'
-          org_membership: puppetlabs
-          fail_if_member: 'true'
-          token: ${{ secrets.IAC_COMMUNITY_LABELER }}
+    if: contains(fromJson('["puppetlabs", "puppet-toy-chest"]'), github.repository_owner)
```

Comment:
do we really need `puppet-toy-chest` ?

Reply:
yes, this is directly from https://github.com/puppetlabs/cat-github-actions/blob/main/example/module/labeller.yml